geopandas/_vectorized.py

```diff
@@ -156,9 +158,10 @@ def to_shapely(data):
         return data
 
 
+@doc(type="a np.ndarray[geoms]")
```

Comment:
Correct me if I'm wrong, but using the name `type` here could cause problems if the decorated function happens to use that built-in function.

Reply:
The core implementation of `doc` is using `format`.

The argument `type` and the function `type()` is different.
The argument `type` is local variable, and the function `type()` is a global variable.

This would get troblems.
```python
type = "a np.ndarray[geoms]"
@doc(type=type)
def func():
    ...
```

If there has another good name, that would be great.